discord/iterators.py

```diff
@@ -237,6 +237,61 @@ async def fill_users(self):
                         await self.users.put(User(state=self.state, data=element))
 
 
+class VoteIterator(_AsyncIterator[Union["User", "Member"]]):
+    def __init__(self, message, answer, limit=100, after=None):
+        self.message = message
+        self.limit = limit
+        self.after = after
+        state = message._state
+        self.getter = state.http.get_answer_voters
+        self.state = state
+        self.answer = answer
+        self.guild = message.guild
+        self.channel_id = message.channel.id
+        self.users = asyncio.Queue()
+
+    async def next(self) -> User | Member:
+        if self.users.empty():
+            await self.fill_users()
+
+        try:
+            return self.users.get_nowait()
+        except asyncio.QueueEmpty:
+            raise NoMoreItems()
+
+    async def fill_users(self):
+        # this is a hack because >circular imports<
```

Comment:
Sorry, the comment should be clear
```suggestion
        # import here to prevent circular imports
```